build/config.yaml

```diff
@@ -2,6 +2,12 @@
 version: 1.0
 sidecars:
   default:
+    X_CSI_SPEC_VERSION: v0.2
+    external-attacher: quay.io/k8scsi/csi-attacher:v0.4.2
+    external-provisioner: quay.io/k8scsi/csi-provisioner:v0.4.2
+    driver-registrar: quay.io/k8scsi/driver-registrar:v0.4.2
+
+  ocp-3.10:
```

Comment:
This should not have the snapshotter, according to https://kubernetes.io/blog/2019/01/17/update-on-volume-snapshot-alpha-for-kubernetes/

Reply:
Hmm, yeah - forgot to remove that (fixed in the corresponding k8s sections though )